Backend/Underdogs-Struggling/src/main/java/com/TheTroisMousquetaires/Underdogs/Struggling/dao/EntryRepository.java

```diff
@@ -6,7 +6,6 @@
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.*;
-import java.util.Optional;
```

Comment:
We should try to avoid `import java.util.*;` and import dependenies one by one especially when amount of dependency is not very much so that we could have better compilation performance.

Reply:
Done